frontend/index.html

```diff
@@ -85,27 +76,19 @@
   </head>
 
   <body>
-    <div class="vcom-header">
-      <div class="container-fluid" style="width: 100%">
-        <a href="https://vaadin.com" class="vcom-link">
-          <span class="visually-hidden">Vaadin</span>
-        </a>
-      </div>
-    </div>
+    <div class="vcom-header" id="haas-container"></div>
     <div id="outlet"></div>
 
     <script>
-      // Loading CSS here so they don't block the bundle download
-      // TODO: remove duplicate stylesheets when taking vaadin.com Header-as-a-service (HaaS) into use
+      // Loading CSS here so they don't block the bundle download. Initiating loading of these
+      // before HaaS, to avoid worse FOUC
       const designSystemCSS = [
-        "https://vaadin.com/frontend/design-system/src/assets/css/1-foundation/custom-properties.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/1-foundation/reset.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/1-foundation/layout.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/2-components/fields.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/2-components/links.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/2-components/tag.css",
-        "https://cdn2.hubspot.net/hubfs/1840687/Design%20System/icons/css/line-awesome.min.css",
-        "https://vaadin.com/frontend/design-system/src/assets/css/1-foundation/typography.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/1-foundation/custom-properties.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/1-foundation/reset.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/1-foundation/typography.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/1-foundation/layout.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/2-components/fields.css",
+        "https://cdn.vaadin.com/vaadin-design-system/latest/assets/css/2-components/tag.css",
```

Comment:
For production, we should look into getting a minified bundled version of these to speed up things.

Reply:
Yeah, sure. I heard @paulroemer might be doing something like that. Can you forward your needs to him?

Reply:
There’s a haas-bundle now, but we still need to load fields.css in addition.